### PR TITLE
Add started guides

### PR DESCRIPTION
**Pull request by trongnhan1312400**
Signed-off-by: nhan.trong.nguyen <nhan.trong.nguyen@logigear.com>

### CODE REVIEW

File: samples/started-guides/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Indy SDK getting started guide
```
Comment (vimmerru) [inline samples/started-guides/README.md]: I suggest to don't use "Getting Started Guide" title as we already have "Getting Started Guide" as a dedicated artifact

File: samples/started-guides/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+The following are scripts using the SDK and the Python wrapper.
```
Comment (vimmerru) [inline samples/started-guides/README.md]: also i am not sure that it is a right place to put this files.

Comment (dhh1128) [reply samples/started-guides/README.md]: @vimmerru I have a different idea about where to put this. I'll talk to you.

**Comment by StevenL2015** [comment]
Any progress on this pull request?

**Comment by vimmerru** [comment]
> Any progress on this pull request?

Raised questions are still opened. So we can't merge it.

**Pull request merged**